_layouts/default.html

```diff
@@ -63,6 +63,60 @@
         <script defer src="/js/bootstrap.min.js"></script>
         <script defer src="/js/scripts.js"></script>
 
+        <!-- Schema.org JSON-LD Markup for Search Engine Optimization -->
+        <script type='application/ld+json'>
+        {
+            "@context": "https://schema.org/",
+            "@type": "SoftwareApplication",
+            "applicationCategory": "FinanceApplication",
+            "name": "Bisq",
+            "image": "https://bisq.network/images/bisq-logo.svg",
+            "availableOnDevice": [
```

Comment:
Bisq isn't available on mobile devices right now...just desktop. There is a notifications app, but I don't think it's right to classify that as Bisq "being available" on those platforms.

Reply:
I wasn't sure if insert the mobile versions or not. Perhaps it is better to remove them.